src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -135,6 +144,19 @@ private void SetFilterHosts(object obj)
         }
 
         #region Properties & Fields
+        internal List<string> addedFields;
+        internal DirectoryReader dirReader;
+        internal Lucene.Net.Store.Directory indexDir;
+        internal IndexWriter writer;
+
+        // Holds the instance for the IndexSearcher
+        internal IndexSearcher Searcher;
+
+        // Used for creating the StandardAnalyzer
+        internal Analyzer Analyzer;
+
+        // Lucene search view model.
+        internal LuceneSearchViewModel LuceneSearchViewModel { get; set; }
```

Comment:
ViewModel are usually linked with a view, since we have no plan here, maybe consider using a Model class only or write as utility class

Reply:
Yeah, converted it to LuceneSearchUtility